### PR TITLE
NEPT-2038: Add wysiwyg templates for file entity and file container

### PR DESCRIPTION
**Pull request by gillesdeudon**
### Description

Insert wysiwyg templates for file entity and file container in order to remove the unwanted "div" container appearing around any files inserted in a WYSIWYG field like "body".

### Change log

- Added: templates/container/container--file--wysiwyg.tpl.php
- Added: templates/file/file--audio--wysiwyg.tpl.php
- Added: templates/file/file--document--wysiwyg.tpl.php
- Added: templates/file/file--image--wysiwyg.tpl.php
- Added: templates/file/file--video--wysiwyg.tpl.php

### Commands

drush cc all

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ env:
   global:
     - DOCKER_COMPOSE_VERSION=1.19.0
     - PATH=$PATH:/home/travis/.composer/vendor/bin:$TRAVIS_BUILD_DIR/vendor/bin
+    - COMPOSER_MEMORY_LIMIT=2G
```
Comment (drupol) [inline .travis.yml]: Unrelated changes, this should have been tackled in another ticket but, ok let's not aim for something perfect.

So, I've rebased the branch properly to have 2 commits:
- 1 for the change in the Travis configuration
- 1 for the template updates

File: templates/container/container--file--wysiwyg.tpl.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+ * Template to display an form container used inside a WYSIWYG field.
+ *
+ * It diverts from the theme_container by displaying its content without any
+ * additional DIV container.
+ *
+ * It is implemented for NEPT-2038.
+ */
+?>
+
+<?php print render($element['#children']); ?>
```
Comment (drupol) [inline templates/container/container--file--wysiwyg.tpl.php]: We should not specify a specific key to render. We should keep `render($element);` in there and remove the `theme wrappers` in a proper preprocess.

**Comment by drupol** [comment]
I've reworded your last commit and replaced **OPENEUROPA-2038** with **NEPT-2038**.

**Comment by imanoleguskiza** [comment]
Thank you for the rewording @drupol ! Can you also please re-review your requests and if they are fine now approve the PR?
